packages/frontend/web-ui/src/game/models/CreateNewGameResult.ts

```diff
@@ -0,0 +1,18 @@
+import { CreateNewGameStatus } from './CreateNewGameStatus';
+import { FormFieldsNewGame } from './FormFieldsNewGame';
+import { FormValidationNewGameResult } from './FormValidationNewGameResult';
+import { GameOptions } from './GameOptions';
+
+export interface setFormFieldsParams {
+  name: string;
+  value: string | number | GameOptions;
+}
+
+export interface CreateNewGameResult {
+  formFields: FormFieldsNewGame;
+  status: CreateNewGameStatus;
+  notifyFormFieldsFilled: () => void;
+  formValidation: FormValidationNewGameResult;
+  backendError: string | null;
+  setFormField: (params: setFormFieldsParams) => void;
```

Comment:
This approach allows setting form fields with a wrong value. We need a better contract here.

Reply:
I've removed the interface SetFormFieldsParams